gems/aws-sdk-core/lib/aws-sdk-core/shared_config.rb

```diff
@@ -45,10 +42,7 @@ def load(credentials_path: nil, config_path: nil)
       # @param profile [String] - the profile this sso session is referenced in.
       # @param sso_session_name [String] - name of the sso_session
       def sso_session(cfg, profile, sso_session_name)
-        # aws sso-configure may add quotes around sso session names
-        # with whitespace
-        sso_session = cfg["sso-session #{sso_session_name}"] ||
-                      cfg["sso-session '#{sso_session_name}'"]
+        sso_session = cfg[sso_session_name]
```

Comment:
Should this still start with "sso-session"?

Reply:
I don't think so. The profile parser now strips out "sso-session" - Based on the SEP, it's supposed to be parsed that way now I think? I don't think sso session names can be the same as profile names? Maybe you can double check me.